Automattic/jetpack
# Payments block: Add transform from buttons

Allow the core/buttons block to be transformed to a jetpack/recurring-payments block

Fixes #17249

##### Limitations
 * It only offers the transform if there is one core/button block inside the core/buttons block.
 * It requires gutenberg >= 11.5.0, the transformation won't be offered in earlier versions. This means using the Gutenberg plug-in or using WordPress >= 5.9


#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Check that you're using WordPress >= 5.9 or have the gutenberg plugin installed and activated.
* Apply patch
* Add a post
* Add button blocks to post
* Select the buttons block (the buttons block contains a button block, so you might have to click up the hierarchy)
* Press the transform/variations button
* Select payments
* You should see the block transformed to a payment block

https://user-images.githubusercontent.com/93301/149837755-acffa11a-2e8e-40a2-925c-65300bb9e728.mp4
